client/src/public/ts/index.ts

```
@@ -483,6 +514,109 @@ function initializeEventListeners() {
+  });
+  $("#archive__search-button").on("click", async () => {
+    const replayID = $("#archive__replay-id").val()?.toString() ?? "";
+    let replayDataJSON;
+    if (replayID in replayCache) {
+      console.log(
+        "Replay data already found in cache, using replay data from cache."
+      );
+      replayDataJSON = replayCache[replayID];
+    } else {
+      console.log("Replay data not found in cache, fetching replay.");
+      let fetchData = await fetchReplay(replayID);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'fetchData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)